package.json

```
@@ -28,34 +28,34 @@
   },
   "dependencies": {
     "@boa-dev/boa_wasm": "^0.20.0",
```

We should update the playground to Boa 0.21:
```suggestion
    "@boa-dev/boa_wasm": "^0.21.0",
```